rhel7/guide.xslt

```diff
@@ -21,27 +21,29 @@
       <xsl:apply-templates select="document('profiles/rht-ccp.xml')" />
       <xsl:apply-templates select="document('profiles/common.xml')" />
-      <xsl:apply-templates select="document('profiles/stig-rhel7-disa.xml')" />
-      <xsl:apply-templates select="document('profiles/stig-rhvh-upstream.xml')" />
-      <xsl:apply-templates select="document('profiles/ospp-rhel7.xml')" />
-      <xsl:apply-templates select="document('profiles/ospp42-draft-rhel7.xml')" />
-      <xsl:apply-templates select="document('profiles/cjis-rhel7-server.xml')" />
-      <xsl:apply-templates select="document('profiles/docker-host.xml')" />
+      <xsl:apply-templates select="document('profiles/disa-stig.xml')" />
+      <xsl:apply-templates select="document('profiles/ospp.xml')" />
+      <xsl:apply-templates select="document('profiles/cjis-server.xml')" />
       <xsl:apply-templates select="document('profiles/nist-800-171-cui.xml')" />
-      <xsl:apply-templates select="document('profiles/stig-ansible-tower-upstream.xml')" />
-      <xsl:apply-templates select="document('profiles/stig-http-disa.xml')" />
-      <xsl:apply-templates select="document('profiles/stig-ipa-server-upstream.xml')" />
-      <xsl:apply-templates select="document('profiles/stig-satellite-upstream.xml')" />
-      <xsl:apply-templates select="document('profiles/anssi_nt28_minimal.xml')" />
-      <xsl:apply-templates select="document('profiles/anssi_nt28_intermediary.xml')" />
-      <xsl:apply-templates select="document('profiles/anssi_nt28_enhanced.xml')" />
-      <xsl:apply-templates select="document('profiles/anssi_nt28_high.xml')" />
       <xsl:apply-templates select="document('profiles/hipaa.xml')" />
 
+      <!-- Profiles currently in draft that should not be included in builds.
+                      Enable/disable locally until profile finalize. -->
+      <!-- <xsl:apply-templates select="document('profiles/docker-host.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/ospp42-draft-rhel7.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/c2s-docker.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-rhvh-upstream.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-ansible-tower-upstream.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-http-disa.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-ipa-server-upstream.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-satellite-upstream.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/anssi_nt28_minimal.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/anssi_nt28_intermediary.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/anssi_nt28_enhanced.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/anssi_nt28_high.xml')" /> -->
```

Comment:
this won't do anything in the new build system. I am surprised this file is even still in there